shared/components/NextConferenceEvent/index.js

```diff
@@ -104,5 +104,6 @@ const NextConferenceEvent = ({ calendarURL, speakers, tickets }) => (
 NextConferenceEvent.propTypes = {
   calendarURL: PropTypes.string,
   speakers: PropTypes.arrayOf(speakerType),
+  tickets: PropTypes.arrayOf(PropTypes.shape(ticketType)),
 };
```

Comment:
Don't think you need this line? You've not changed the NextConferenceEvent component and I think it generates a props warning when viewing the conference page